nautobot_design_builder/tests/test_design_job.py

```diff
@@ -194,8 +191,35 @@ def setUp(self):
     def test_create_integration_design(self):
```

Comment:
Is this now the failing test that demonstrates I need to fix the cable and next prefix extensions?

Reply:
no, the ones that fail have been skipped with `@unittest.skip` until are fixed (below)